### ISSUE TITLE
fCC Everything Be True Challenge --> Missing Closing Quote In Test Cases Shown

### ISSUE THREAD

**Issue by jfines** [freeCodeCamp/freeCodeCamp#45455]
## https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/everything-be-true

**The test cases are missing a closing quote in what is displayed in the challenge and there is no way to tell if it's the same under the hood or not - probably not because the code won't even run the way it is shown in the challenge.**

```
truthCheck([{id: 1, data: {url: "https://freecodecamp.org", name: "freeCodeCamp}}, {id: 2, data: {url: "https://coderadio.freecodecamp.org/", name: "CodeRadio"}}, {id: null, data: {}}], "data") should return true.

truthCheck([{id: 1, data: {url: "https://freecodecamp.org", name: "freeCodeCamp}}, {id: 2, data: {url: "https://coderadio.freecodecamp.org/", name: "CodeRadio"}}, {id: null, data: {}}], "id") should return false.
```

**The word "freeCodeCamp"  in those examples has a " in front of it but not after it - causing the code to fail.**

**What is the proper thing to do? Just fix it?**

- - -    

      **Your browser information:**

User Agent is: <code>Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:98.0) Gecko/20100101 Firefox/98.0</code>

**Challenge:**  Everything Be True

**Link to the challenge:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/everything-be-true

```

## Expected behavior

<!-- Add a clear and concise description of what you expected to happen. -->

When the test cases are used to develop a solution to the challenge the code fails. Please add a closing quote to the last two test cases shown on the challenge page so that they may be used directly in our efforts to develop a solution to the challenge.

## Screenshots

<!-- If applicable, add screenshots to help explain your problem. you can drag and drop, png, jpg, gif, etc. in this box. -->

![Screenshot 2022-03-17 at 22-28-59 freeCodeCamp org](https://user-images.githubusercontent.com/86216502/158943232-e4e707cf-524b-4b77-bce7-b41df248a3e7.png)

## System

<!-- Please complete the following information. -->

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

## Additional context

<!-- Add any other context about the problem here. -->

**Comment by gikf** [comment]
Thanks for opening this issue.

This looks something that can be fixed by "first time" code contributors to this repository. Here are the files that you should be looking at to work on a fix:

List of files:

1. https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/everything-be-true.md

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.

**Comment by anjiqbal** [comment]
Hi @gikf , I'm new to open source and would love to contribute to this

**Comment by gikf** [comment]
@anjiqbal :+1:, no need to ask for permission. Issues labelled with `first timers only` or `help wanted` are open for contributions. My previous comment includes couple of links that should help with getting started and if you'd get stuck.

**Comment by Sembauke** [comment]
Hey @anjiqbal,
It seems you have opened a pr on your own version of the repository.

Do you mind opening it here instead?

Have a nice day and happy coding 😄

**Comment by gikf** [comment]
Closed with #45460
